frontend/src/component/project/Project/Project.tsx

```diff
@@ -180,7 +181,8 @@ export const Project = () => {
         },
         {
             title: 'Project settings',
-            path: `${basePath}/settings${isOss() ? '/environments' : ''}`,
+            path: `${basePath}/settings`,
+            ossPath: `${basePath}/settings/api-access`,
```

Comment:
Introducing ossPath, because oss does not have default /settings path, but they start from api-access.